spec/classes/lightblue/application/datamgmt_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'lightblue::application::datamgmt' do
+  let(:hiera_config){ 'spec/fixtures/hiera/hiera.yaml' }
+
+  let :facts do
+    {
+      :architecture => 'x86_64',
+      :osfamily => 'RedHat',
+      :operatingsystemrelease => '7.0'
+    }
+  end
+
+  context 'with default params' do
+    it do
+      should contain_package('lightblue-data-mgmt') \
+        .with_ensure('latest') \
+        .with_require(['Class[Lightblue::Yumrepo::Lightblue]', 'Class[Lightblue::Eap]'])
+    end
+
+    it do
```

Comment:
I am fairly new to spec tests myself so hardly an expert, but also a fan of consistency. Our spec tests are still new, so changing them at this point wouldn't be a major undertaking.

When I write unit tests in other languages, I intentionally keep the test cases as simple as possible for two reasons: one, readability, and two, so that any failure is obvious. That said, it sounds like what @alechenninger is purposing would make it easier to identify what failed. So I am fine with switching, especially if that is the generally accepted "best practice".